src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -1416,6 +1416,38 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         <Extension>
+        Public Function GetInConversion(compoundOperation As ICompoundAssignmentOperation) As Conversion
```

Comment:
The parameter name is `compoundAssignment` in C# extension methods and `compoundOperation` here. Consider using the same name.